apps/taxes/src/modules/taxjar/taxjar-order-created-adapter.ts

```diff
@@ -0,0 +1,105 @@
+import { CreateOrderRes, LineItem } from "taxjar/dist/util/types";
+import { OrderCreatedSubscriptionFragment } from "../../../generated/graphql";
+import { ChannelConfig } from "../channels-configuration/channels-config";
+import { numbers } from "../taxes/numbers";
+import { CreateOrderResponse } from "../taxes/tax-provider-webhook";
+import { WebhookAdapter } from "../taxes/tax-webhook-adapter";
+import { CreateOrderArgs, TaxJarClient } from "./taxjar-client";
+import { TaxJarConfig } from "./taxjar-config";
+import { taxProviderUtils } from "../taxes/tax-provider-utils";
+
+type Payload = { order: OrderCreatedSubscriptionFragment; channel: ChannelConfig };
+type Target = CreateOrderArgs;
+type Response = CreateOrderResponse;
+
+export function sumPayloadLines(lines: LineItem[]): number {
```

Comment:
isn't it some generic utility? (now its inside taxjar)

Reply:
nope, previously it was also inside taxjar